packages/hint-compat-api/src/helpers/compat-api.ts

```diff
@@ -107,15 +103,13 @@ export class CompatApi {
 
         // Sometimes the API returns an array but only the first seems relevant
         if (Array.isArray(currentBrowserFeatureSupported) && currentBrowserFeatureSupported.length > 0) {
-            if (prefix) {
-                currentBrowserFeatureSupported = currentBrowserFeatureSupported.find((info) => {
-                    return info.prefix === prefix;
+            currentBrowserFeatureSupported = currentBrowserFeatureSupported
```

Comment:
Why are you using `filter` and `find`? I think you should be able to do everything inside `find`.

Reply:
In both cases, I was trying to increase the readability of the code splitting the responsibility per operator. Of course, this is against performance. Probably, in that case, the performance is key so I will remove one of the operators.

Reply:
I closed the previous conversation since the issue seems similar.